src/components/Container.js

```diff
@@ -48,4 +49,6 @@ class Container extends Component {
     }
 }
 
-export default Container;
+export default connect(state => ({
```

Comment:
Please, @chmieludaniel keep the same format of mapping state to props. Use mapStateToProps function with fallback and propTypes validation.